src/Exports/DonorsExport.php

```diff
@@ -91,6 +98,22 @@ public function get_data(): array
             }
         }
 
+        /**
+         * Filter by donation form
+         *
+         * @unreleased
+         */
```

Comment:
Can we move the docblock up to `get_data()` please

Reply:
Resolved https://github.com/impress-org/givewp/pull/7146/commits/d99d6cf5e7ef235e94909da13b1792212bb1722a